WORKSPACE

```diff
@@ -39,8 +39,8 @@ git_repository(
 
 git_repository(
     name = "graknlabs_build_tools",
-    remote = "https://github.com/graknlabs/build-tools",
-    commit = "4815b8583ca98ad59058d1a808b87e6b2f47dbf7", # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_build_tools
+    remote = "https://github.com/lolski/build-tools",
```

Comment:
Make sure to merge the PR on `build-tools` first, then update this to the real `build-tools` repo, before merging this PR.